merkletree/tb.go

```diff
@@ -1,17 +1,26 @@
 package merkletree
 
-import "github.com/coniks-sys/coniks-go/crypto/sign"
+import (
+	"bytes"
+
+	"github.com/coniks-sys/coniks-go/crypto/sign"
+)
 
 type TemporaryBinding struct {
-	Index     []byte
-	Value     []byte
-	Signature []byte
+	// IssuedEpoch would not be included in the serialization.
+	// To verify the validation of IssuedEpoch, we can compare it
+	// against the epoch of STR included in the serialization.
```

Comment:
I'm assuming the validation you're referring to here is the client verifying that `IssuedEpoch == STR.Epoch+1` when it receives the TB?